npm/iptm/iptm.go

```diff
@@ -114,11 +114,65 @@
 		}
 	}
 
+	// Insert a RETURN on MARK rule for INGRESS in in AZURE-NPM-INGRESS-PORT chain
+	entry.Chain = util.IptablesAzureIngressPortChain
+	entry.Specs = []string{
+		util.IptablesJumpFlag,
+		util.IptablesReturn,
+		util.IptablesModuleFlag,
+		util.IptablesMarkVerb,
+		util.IptablesMarkFlag,
+		util.IptablesAzureIngressMarkHex,
+		util.IptablesModuleFlag,
+		util.IptablesCommentModuleFlag,
+		util.IptablesCommentFlag,
+		"RETURN-on-INGRESS-mark-0x2000",
```

Comment:
nit: can we use fmt.Sprintf("RETURN-on-INGRESS-mark-+v",IptablesAzureIngressMarkHex) to make sure the comments reflect the mark being added?